includes/REST/DummyDataController.php

```diff
@@ -50,9 +51,9 @@ public function register_routes() {
                     'methods'             => WP_REST_Server::CREATABLE,
                     'callback'            => array( $this, 'import_dummy_data' ),
                     'permission_callback' => array( $this, 'get_permissions_check' ),
-                    'args'                => $this->get_endpoint_args_for_item_schema( WP_REST_Server::CREATABLE ),
```

Comment:
Previous implementation was ok, kindy discuss with me before you work on this PR.